### PR TITLE
ci: remove unused artifact from e2e

### PR DESCRIPTION
**Pull request by lionelB**
remove artifact from e2e job

### CODE REVIEW

**Review by UnbearableBear — APPROVED** [review]

File: .gitlab-ci/stages/e2e.yml [hunk 1]
```diff
@@ -9,11 +9,6 @@ Test on deployed branch:
     variables:
       # Don't run when deploying in production
       - $PRODUCTION
-  artifacts:
-    expire_in: 1 day
-    when: always
-    paths:
-      - optional/e2e/runners/puppeteer/output
```
Comment (douglasduteil) [inline .gitlab-ci/stages/e2e.yml:16]: Is it really a problem ?
Those artifacts are the only visual hint we have today when a e2e test fails.

Comment (lionelB) [reply .gitlab-ci/stages/e2e.yml:16]: Well, I have no idea how to retrieve that. Also A day artifact is too short, most of the time we have PR that live a couple of days

File: .gitlab-ci/stages/e2e.yml [hunk 2]
```diff
@@ -26,6 +21,14 @@ Test on deployed branch:
     - source ./.gitlab-ci/env.sh
     - CODECEPT_BASEURL=${FRONTEND_URL}
     - cd ./optional/e2e/runners/puppeteer
+    - |-
+      retry=60;
+      while
+        ! curl -sS "${FRONTEND_URL}/api/version" | grep "${COMMIT}" &&
```
Comment (douglasduteil) [inline .gitlab-ci/stages/e2e.yml]:
```suggestion
        ! wget -qO - "${FRONTEND_URL}/api/version" | grep "${COMMIT}" &&
```

File: .gitlab-ci/stages/deploy.yml [hunk 3]
```diff
@@ -122,26 +120,11 @@ Deploy @cdtn/frontend (dev):
...
-      - packages/code-du-travail-nlp
-  after_script:
-    - |-
-      source ./.gitlab-ci/env.sh
-      retry=60;
-      while
-        ! curl -sS "${FRONTEND_URL}/api/version" | grep "${COMMIT}" &&
-        [[ $(( retry-- )) -gt 0 ]];
-      do echo "Waiting for frontend to be ready" ; sleep 3 ; done ;
-      [ "$retry" -eq "-1" ] && exit 1
```
Comment (douglasduteil) [inline .gitlab-ci/stages/deploy.yml:144]: this feels of topic of the pr non ?
could it be merged in a separate pr ?

Comment (lionelB) [reply .gitlab-ci/stages/deploy.yml:144]: once we will a robust ci, I will do micro PR

File: .gitlab-ci/stages/e2e.yml (hunk 1 above)
Comment (douglasduteil) [reply .gitlab-ci/stages/e2e.yml:16]: > A day artifact is too short, most of the time we have PR that live a couple of days

Change it to 1 week maybe

File: .gitlab-ci/stages/quality.yml [hunk 4]
```diff
@@ -98,7 +98,7 @@ Test @cdtn/api:
       echo $ELASTICSEARCH_URL;
       retry=60;
       while
-        ! curl -sS "$ELASTICSEARCH_URL/_cat/health?h=status" &&
+        ! wget -qO - "$ELASTICSEARCH_URL/_cat/health?h=status" &&
```
Comment (douglasduteil) [inline .gitlab-ci/stages/quality.yml]:
```suggestion
        ! wget -q --spider "$ELASTICSEARCH_URL/_cat/health?h=status" &&
```

`-O -` is to pipe the result to something
